main/samba/src/EBox/Samba/User.pm

```diff
@@ -436,6 +436,7 @@ sub create
     my ($class, %args) = @_;
 
     # Check for required arguments.
+    throw EBox::Exceptions::MissingArgument('name') unless ($args{name});
```

Comment:
I don't see in the description the name param neither in `EBox::Samba::CGI::AddUser` this parameter. Could you put a shed a light on here @javierag?